account_financial_report/report/templates/general_ledger.xml

```diff
@@ -352,7 +352,41 @@
                                 </span>
                             </t>
                         </div>
-                        <div class="act_as_cell amount" style="width: 3.63%;" />
+                        <div class="act_as_cell amount" style="width: 3.63%;">
+                            <t t-if="type == 'account_type'">
+                                <t
+                                    t-set="domain"
+                                    t-value="[('account_id', '=', account['id']),
+                                             ('date', '&lt;', o.date_from)]"
+                                />
+                                <span
+                                    t-att-domain="domain"
+                                    res-model="account.move.line"
+                                >
+                                    <t
+                                        t-raw="account_or_group_item_object['init_bal']['bal_curr']"
+                                        t-options="{'widget': 'monetary', 'display_currency': account['currency_id']}"
+                                    />
+                                </span>
+                            </t>
+                            <t t-if="type == 'grouped_type'">
+                                <t
+                                    t-set="domain"
+                                    t-value="[('account_id', '=', account['id']),
+                                             ('partner_id', '=', group_item['id']),
```

Comment:
Although not of this PR, you can also group by tax, so this is not correct.

Reply:
And why not sharing certain parts of the domain for the whole line?

Reply:
I have added an initial commit to reduce code and re-use domain.